GSA/digitalgov.gov
# Update CX to Customer Experience

## Summary

This PR will update any reference of `CX` to `Customer Experience` and remove the topic `Customer Experience Community of Practice` and merge into `Customer Experience` 

The ticket can be viewed by searching for the title `Update existing tag on https://digital.gov/topics/cx/ from "cx" to "customer-experience"` within the tracking system 

### Preview

View testing steps below for preview links



### Solution
Updated any old references and changed the `cx` slug into the new `customer-experience` 

### How To Test

1. Vist the new [Customer Experience](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cx-udpate-sandbox/topics/customer-experience/) topic page and check that the URL is no longer `/topics/cx/ and also check [Topics Page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cx-udpate-sandbox/topics/) and check that CX is not longer listed 
2. Review files changed and verify that CX is no longer referenced

---

### Dev Checklist


Tagging supervisor - @rayestrada
